internal/merge/merge.go

```diff
@@ -2,11 +2,11 @@
 // Use of this source code is governed by an Apache2
 // license that can be found in the LICENSE file.
 
-package loader
+package merge
```

Comment:
Nit: Most OPA packages have a doc comment that explains what the package is for. https://godoc.org/github.com/open-policy-agent/opa

Reply:
Good call, I got lazy since its internal.. but we should be consistent about that sort of thing.